bsp/worker/src/mill/bsp/worker/MillBuildServer.scala

```diff
@@ -729,7 +676,7 @@ private class MillBuildServer(
             }
         }
 
-      agg(evaluated.flatten.toSeq.asJava)
```

Comment:
Could we keep the signature of `completableTasks`, and add a new method `completableTasksWithState` (or something like that) that calls `agg` with two parameters? `completableTasks` can just be a thin wrapper around `completableTasksWithState`, but this would keep us from having to add boilerplate to all the callsites, the vast majority of which do not care about `State`